src/components/BorrowerProfile/SupporterDetails.vue

```diff
@@ -1,41 +1,67 @@
 <template>
 	<div class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center">
-		<!-- eslint-disable-next-line max-len -->
-		<div class="tw-block md:tw-inline-block md:tw-flex-none md:tw-mr-2 tw-w-[135px] md:tw-w-[150px] lg:tw-w-[84px] xl:tw-w-[96px]">
-			<borrower-image
-				class="
-					tw-w-full
-					tw-bg-black
-					tw-rounded
-				"
-				:alt="name"
-				:aspect-ratio="borrowerImageAspect"
-				:default-image="{ width: isMobile ? 160 : 96 }"
-				:hash="hash"
-				:images="[
-					// small & medium
-					{ width: 180, viewSize: 414 },
-					// large
-					{ width: 84, viewSize: 734 },
-					// xlarge
-					{ width: 96, viewSize: 1024 }
-				]"
-			/>
-		</div>
-		<div class="tw-inline-block">
-			<span
-				v-if="name"
-				:class="`${ displayType === 'teams' ? 'tw-text-h4' : ''}`"
-			>
-				{{ name }}
-			</span><br>
-			<span
-				v-if="lenderPage && whereabouts"
-				class="tw-text-h4"
-			>
-				{{ whereabouts }}
-			</span>
-		</div>
+		<router-link
+			class="tw-contents tw-no-underline"
```

Comment:
I believe you could remove the div that is the parent of this, and replace `tw-contents` with all of the classes from that div.

Reply:
Then if you wanted black text, you could add a class `tw-text-black` to this.